src/Uno.UI/UI/Xaml/Controls/Primitives/ToggleButton.cs

```diff
@@ -70,5 +64,27 @@ public void OnTemplateRecycled()
 		{
 			IsChecked = false;
 		}
+
+		protected virtual void OnToggle()
```

Comment:
OnAutomationToggle()? Is it used/useful in any other context?

Reply:
`OnToggle` is part of `ToggleButton`'s contract.

`AutomationPeerToggle` exists to allow `OnToggle` to be called from automation peers.